Uplift/Controllers/GymDetailViewController.swift

```diff
@@ -151,7 +150,15 @@ extension GymDetailViewController: UICollectionViewDataSource, UICollectionViewD
 
         switch itemType {
         case .hours:
-            return CGSize(width: width, height: GymDetailHoursCell.baseHeight)
+            let baseHeight = CGFloat(GymDetailHoursCell.Constants.hoursTitleLabelTopPadding +
```

Comment:
could we either do the `baseHeight` calculations for each itemType in this VC, or move all of the `baseHeight` calculations to the respective cells?

Reply:
I made an issue for this